build-release-tools/lib/RepositoryOperator.py

```diff
@@ -85,6 +86,7 @@ def run_git_command(git, command, directory, results):
                              'return_code': return_code
                             })
 
+    @retry(Exception)
```

Comment:
does every 'do_one_task' could 'retry' ?  it woundn't impact the whole process, or maybe it's not proper in some cases?